src/utils/componentMixin.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+'use strict';
+
+/**
+ * Mixin the attributes and functions to target's prototype
+ * @param {Object} target React component
+ * @param {Object} source the mixin source
+ * @method componentMixin
+ */
+module.exports = function componentMixin(target, source) {
```

Comment:
can add class static properties here as well?  like `componentMixin(target, protoProps, staticProps)`  Noticed when you use `componentMixin()` later, you almost always mix in both prototype props and static props.